### DISCUSSION TITLE
[Question]: How to train and test on my own dataset?

### DISCUSSION THREAD

**Discussion by lyj-fdu** [open-edge-platform/anomalib#2420] [Q&A] [answered]

**Comment by shanmugamani1023** [comment, marked as answer]
follow readme file of this repo https://github.com/shanmugamani1023/anomalib_pattern_recognition

**Comment by samet-akcay** [comment]
There is also a dedicated documentation for this below
https://anomalib.readthedocs.io/en/v1.2.0/markdown/guides/how_to/data/custom_data.html
